src/text.ts

```diff
@@ -1,24 +1,33 @@
-export function insertText(
-  textarea: HTMLInputElement | HTMLTextAreaElement,
-  text: string,
-  options = {addNewline: true}
-): void {
-  const beginning = textarea.value.substring(0, textarea.selectionStart || 0)
-  const remaining = textarea.value.substring(textarea.selectionEnd || 0)
+export function insertText(textarea: HTMLInputElement | HTMLTextAreaElement, text: string): void {
+  const before = textarea.value.slice(0, textarea.selectionStart || undefined)
+  const after = textarea.value.slice(textarea.selectionEnd || undefined)
```

Comment:
The ` || undefined` additions prevent `Argument of type 'number | null' is not assignable to parameter of type 'number | undefined'.ts(2345)` errors. I could not think of another solution.

Reply:
Might be safer to use nullish coalescing to avoid falling back to `undefined` if the value is `0`.
```suggestion
  const before = textarea.value.slice(0, textarea.selectionStart ?? undefined)
  const after = textarea.value.slice(textarea.selectionEnd ?? undefined)
```

Reply:
@dgreif Perfect! I changed it in fa969951f4dc64603e4658e46f13e095fe554bce.